integration/data/cmd.py

```diff
@@ -57,10 +58,25 @@ def snapshot_purge(url=CONTROLLER):
     return subprocess.check_call(cmd)
 
 
+def backup_status(backupID, url=CONTROLLER):
+    output = ""
+    cmd = [_bin(), '--url', url, 'backup', 'status', backupID]
+    for x in range(3600):
```

Comment:
Can we use `common.RETRY_COUNTS`? The backup size in our integration test is limited hence we don't need to retry 3600 times.